client/new-stack-editor/lib/controllers/base.coffee

```diff
@@ -0,0 +1,10 @@
+kd = require 'kd'
+
+
+module.exports = class BaseController extends kd.Object
+
+  constructor: (options = {}, data) ->
+
+    super options, data
+
+    @editor = @getOption 'editor'
```

Comment:
Right now this only creates just another indirection with any visible gain (`@getOption 'editor'` is always available without this base class), but i am guessing you are thinking about extending this one's functionality.